product_esc_code/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module does the following:
+- Give the product an ESC code so that it can be searched by ESC code.
```

Comment:
```suggestion

* Adds the alternative code field to the product, which should be part of the product display name presentation and can also be used in product search.
```
We need a blank line before bullet points (or the presentation will be distorted).